tests/integration/components/aria-tabs-test.js

```diff
@@ -51,15 +51,16 @@ module('Integration | Component | aria-tabs', function (hooks) {
     assert.equal(tabs.length, 4);
     assert.equal(tabPanels.length, 4);
     assert.equal(disabledTab.length, 1);
```

Comment:
https://github.com/simplabs/qunit-dom/blob/master/API.md#exists
Tu peux lui passer le nombre désiré

Reply:
Et du coup, même réflexion à chaque `assert.equal(list.length, N)`

Reply:
Not relevant here as we need to tests individual elements in following lines